Controller/CRUDController.php

```diff
@@ -234,7 +234,8 @@ public function editAction($id = null)
         // the key used to lookup the template
         $templateKey = 'edit';
 
-        $id = $id ?: $request->get($this->admin->getIdParameter());
+        // If the $id variable is provided AND the child ID is not needed, do not reload it.
```

Comment:
I read the commit comment and get it now, but this could be made a bit more clear in the code itself IMO